ui-tests/src/test/java/io/syndesis/qe/steps/other/GoogleCalendarSteps.java

```diff
@@ -0,0 +1,36 @@
+package io.syndesis.qe.steps.other;
+
+import cucumber.api.java.en.And;
+import io.syndesis.qe.pages.integrations.editor.add.connection.actions.google.CreateEvent;
+import io.syndesis.qe.pages.integrations.editor.add.connection.actions.google.GetSpecificEvent;
+import io.syndesis.qe.utils.GoogleCalendarUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class GoogleCalendarSteps {
+    @Autowired
+    private GoogleCalendarUtils gcu;
+
+    @And("^fill in get specific event form using account \"([^\"]*)\" calendar \"([^\"]*)\" and event \"([^\"]*)\"$")
```

Comment:
Using @And seems like auto-generated code, would it not make more sense to use @When ? (I'm aware it makes no difference)